### ISSUE TITLE
Permalink feature

### ISSUE THREAD

**Issue by slrslr** [zer0net/zerovoat#136]
Hello,

can it be useful to add permanent link to each comment so it can be easily linked from other pages or zites?

Maybe the link can also be added to /index.html?v=user-admin+s=comments

Because i am unable to enter the topic or be redirected to that topic/question.

**Comment by haschimoto** [comment]
You mean there should be an anchorpoint to each comment to directly jump to it when linked from anywhere else?

**Comment by slrslr** [comment]
Yes, some point where one can right click and copy link of the comment. And when i someone land to that page, it will scroll to the comment. ZeroTalk added tis recently. It can be found when you hover the comment and in its top right corner are three dots.
